### PR TITLE
I created a route image.

### PR DESCRIPTION
**Pull request by Sonya092**

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    "statusBarItem.hoverBackground": "#2f7c47",
+    "statusBarItem.remoteBackground": "#215732",
+    "statusBarItem.remoteForeground": "#e7e7e7",
+    "titleBar.activeBackground": "#215732",
+    "titleBar.activeForeground": "#e7e7e7",
+    "titleBar.inactiveBackground": "#21573299",
+    "titleBar.inactiveForeground": "#e7e7e799"
+  },
+  "peacock.color": "#215732"
+}
```
Comment (KeerhemFR) [inline .vscode/settings.json]: You shouldn't send your vscode settings

File: routes/images.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
+const express = require('express');
+const { db } = require('../conf');
+
+const router = express.Router();
+
+router.get('/', async (req, res) => {
+  try {
+    const [images] = await db.query(`SELECT id, url, title FROM images`);
```
Comment (KeerhemFR) [inline routes/images.js]: There isn't a title column in db, and the route miss the description and pagesId column

File: routes/images.js [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+  } catch (err) {
+    res.status(500).send('Error retrieving the images');
+  }
+});
+
+router.get('/:id', async (req, res) => {
+  try {
+    const { id } = req.params;
+    const [images] = await db.query(
+      `SELECT id, url, title FROM images WHERE id = ?`,
```
Comment (KeerhemFR) [inline routes/images.js]: same

File: routes/images.js [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+    }
+  } catch (err) {
+    res.status(500).send('Error retrieving the image');
+  }
+});
+
+router.post('/', async (req, res) => {
+  try {
+    const { url, title } = req.body;
+    await db.query(`INSERT INTO images (url, title) VALUES (?, ?)`, [
```
Comment (KeerhemFR) [inline routes/images.js]: same

**Review by KeerhemFR — CHANGES_REQUESTED** [review]
Resume of the needed change :
- No need to commit the .vscode settings
- The routes queries doesn't match the database
- The Images routes miss a put route to update the images

**Review by KeerhemFR — APPROVED** [review]
See no evil here

**Review by Hacene010 — APPROVED** [review]

**Pull request merged**